workers/unity/Assets/Playground/Scripts/Player/PlayerCommandsSystem.cs

```diff
@@ -102,9 +106,10 @@ protected override void OnUpdate()
                         command == PlayerCommand.LaunchLarge ? LargeEnergy : SmallEnergy,
                         playerId
                     ));
-
+#if UNITY_EDITOR
                 lastId = commandSystem.SendCommand(request, entities[0]);
```

Comment:
we still want to send the command, it's only the logging we don't really care about